assets/javascripts/discourse/templates/components/translate-topic-title.hbs

```diff
@@ -0,0 +1,4 @@
+<a {{action "translate"}} title={{i18n "translator.view_translation"}} class="translate-title" role="button">
```

Comment:
I'm curious why a link tag is used here instead of a button because I believe using a button here will work.